test/RuleSet.test.js

```diff
@@ -169,5 +169,92 @@ describe("testRuleSet", () => {
         rule = ruleset.getRule("resource-icu-plurals");
         expect(rule).toBe(rule2);
     });
+
+    test("RuleSet remove a rule", () => {
```

Comment:
there should be some tests which show interactions between ruleset definitions so that it's clear what behaviour is expected when a rule is `true` in one set but not defined in another, or `true` in one but `false` in another etc.

Reply:
done in FileType.test.js

Reply:
This file does not seem to be modified in current PR

Reply:
Sorry, forgot to push the branch again.